docs/lcms/README.md

```diff
@@ -144,7 +144,7 @@ The specific type of assay being executed.
 
 | constraint | value |
 | --- | --- |
-| enum | `LC-MS (metabolomics)`, `LC-MS/MS (label-free proteomics)`, or `MS (shotgun lipidomics)` |
+| enum | `TMT LC-MS`, `Targeted Shotgun / Flow-injection LC-MS`, or `Untargeted LC-MS` |
```

Comment:
Should be just 3 mass spec types:
-Labeled LC-MS/MS
-Label-free LC-MS/MS
-Label-free MS/MS

The omics types are indicated in the _analyte_class_ field: proteins, metabolites or lipids

Reply:
Currently
```
| enum | `Label-free MS/MS`, `Label-free LC-MS/MS`, or `Labeled LC-MS/MS` |
```